lib/vercel/schemas.ts

```diff
@@ -746,3 +746,32 @@ export const TransferInstallationToMarketplaceRequestSchema = z.object({
       "The account information for this installation. Use Get Account Info API to re-fetch this data post transfer."
     ),
 });
+
+export type RequestTransferFromMarketplace = z.infer<
+  typeof requestTransferFromMarketplaceSchema
+>;
+export const requestTransferFromMarketplaceSchema = z.object({
+  transferId: z.string().min(1),
+  requester: z.object({
+    name: z.string().min(1),
+  }),
+});
+
+export type RequestTransferFromMarketplaceResponse = z.infer<
+  typeof requestTransferToMarketplaceResponseSchema
+>;
+export const requestTransferFromMarketplaceResponseSchema = z.object({
+  continueUrl: z.string().url(),
+});
+
+export type TransferInstallationFromMarketplaceRequest = z.infer<
+  typeof TransferInstallationFromMarketplaceRequestSchema
+>;
+
+export const TransferInstallationFromMarketplaceRequestSchema = z.object({
```

Comment:
Start this const name with lowercase, since that's what you do with others above.